---
title: 「E メール内の承認リンクをクリックすると、エラーが表示されます」
description: 説明
solution: Campaign,Campaign Classic v7
product: Campaign,Campaign Classic v7
applies-to: "Campaign Classic,Campaign Classic v7"
keywords: "KCS,workflow error A technical error has occurred"
resolution: Resolution
internal-notes: null
bug: false
article-created-by: Amol Mahajan
article-created-date: "3/14/2023 3:59:29 PM"
article-published-by: Amol Mahajan
article-published-date: "3/15/2023 10:52:18 AM"
version-number: 3
article-number: KA-19380
dynamics-url: "https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=0aced72d-81c2-ed11-83ff-6045bd0065b6"
source-git-commit: c50cba5c6fe03331df9fada0401be04a90b037dc
workflow-type: tm+mt
source-wordcount: '421'
ht-degree: 1%

---

# E メール内の承認リンクをクリックすると、エラーが表示されます

## 説明 {#description}

<b>環境</b>
- Campaign Classic
- Campaign Classic v7



<b>問題/症状</b><br><br> <br><br>E メール内の承認リンクをクリックすると、次のエラーが表示される場合があります。<br><br>


技術的なエラーが発生しました：

WDB-200001 SQL 文「SELECT W0.iWorkflowId, W0.sInternalName, W0.sLabel, W0.sDesc, W0.iCreatedById, W0.iModifiedById, W0.tsCreated, W0.tsLastModified, W0.iSupervisorId, W0.sTimezone, W0.iInProcess, W0.iHistory, W0.iOnError, W0.iErrorLimit, W0.iBuiltIn, W0.iIsModel, W0.sForm, W0.iFolderId, W0.iFolderProcessId, W0.tsLastStart, W0.tsNextProcessing, W0.sNextProcessingAfn, W0.sNextProcessingHost, W0.iStatus, W0.0iProduction, W0.iFailed, W0.iUnlockedCount, W0.iState, W0.tsProcess, W0.iProcessId, W0.sHostname, W0.iPort, W0.iOperationId, W0.iStartState,W0.iStartState = 1 および W0.iForecasted = 0 および（W0.iState = 0 または W0.iState = 20）その後、W0.iState = 1 および W0.iForecasted = 1 および（W0.iState = 0 または W0.i0）の場合、1 の場合 State = 20) 次に 2 個の W0.iState の終了、W0.iPriority、W0.iForecasted、W0.sLogin、W0.iRoutingOperationId、W0.iOrder、W0.iExpiration、W0.iRecurrentMdl, W0.iCurrentOrderId, W0.mData, W0.mVisualState FROM XtkWorkflow W0 WHERE ((W0.iFolderId IN) (SELECT XR.iFolderId FROM XtkRights WHER.IOpERATORId IN (XXXX, XXXX, XXXX, XXXX) OR W0.iFolderId=0) AND (((W0.iFolderId IN （XtkRights XR で XR.iOperatorId）を選択 ID (XXXX, XXXX, XXXX, XXXX) または W0.iFolderId=0)) AND ((W0.iWorkflowId = :#(1)#)))&#39;を実行できませんでした。

Param(0)=50302570



WDB-200011リクエストされたデータベースレコードは存在しません。

条件「(/@id = XXXXXX)」を満たすタイプ「xtk:workflow」のドキュメントを読み込めません。



サービス「xtk:workflow」の SOAP メソッド「GetValidationParameters」の呼び出し中にエラーが発生しました ):&#39;SOP-330011サービス「xtk:workflow」のメソッド「GetValidationParameters」の実行中にエラーが発生しました。



コード：&#39;SOAP-ENV:Server&#39;

com.neolane.fwk.core.NlException:サービス「xtk:workflow」の SOAP メソッド「GetValidationParameters」の呼び出し中にエラーが発生しました ):&#39;SOP-330011サービス「xtk:workflow」のメソッド「GetValidationParameters」の実行中にエラーが発生しました。

コード：&#39;SOAP-ENV:Server&#39;

at com.neolane.fwk.soap.SoapMethodCall.execute(SoapMethodCall.java:840)

at com.neolane.fwk.xtk.JSPContext.executeSOAPCall(JSPContext.java:820)

org.apache.jsp.validateTask_jsp で、_jspService(validateTask_jsp.java:176)

at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)

at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)

at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:439)

at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:395)

at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:339)

at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)

at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)

at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)

at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)

at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

at com.neolane.jssp.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:102)

at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)

at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)

at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:110)

at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)

at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)

at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)

at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:445)

at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1115)

at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:637)

at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)

at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)

at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)

at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)

at java.lang.Thread.run(Thread.java:748)


## 解決策 {#resolution}


オペレーターは、管理/実稼動/オブジェクトの下にあるキャンペーンワークフローフォルダーにアクセスできる必要があります。
